### PR TITLE
Add standalone API source files + links in README

### PR DESCRIPTION
**Pull request by vvenance**
Added missing source files from the Bump demo org.
File name convention has been followed so the CI should not break

### CODE REVIEW

**Review by paulRbr — COMMENTED** [review]
Looks nice, thanks for this :). Could you add all the token variables needed for each documentation (similarly to [this commit](https://github.com/bump-sh/examples/pull/25/commits/c6c77f060ae77e52f6a813738a5a9d0e3d4043f4)).

Note: we should really start to implement API tokens based on Users (instead of resource) at some point in the product! :slightly_smiling_face:

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -31,3 +31,20 @@ jobs:
...
+          LUMAPPS_BUMP_TOKEN: ${{ secrets.LUMAPPS_BUMP_TOKEN }}
+          MAPOTEMPO_BUMP_TOKEN: ${{ secrets.MAPOTEMPO_BUMP_TOKEN }}
+          NEXTAUTH_BUMP_TOKEN: ${{ secrets.NEXTAUTH_BUMP_TOKEN }}
+          PAPPERS_BUMP_TOKEN: ${{ secrets.PAPPERS_BUMP_TOKEN }}
+          PAYGREEN_BUMP_TOKEN: ${{ secrets.PAYGREEN_BUMP_TOKEN }}
+          REMOVE_BG_BUMP_TOKEN: ${{ secrets.REMOVE_BG_BUMP_TOKEN }}
+          RESTAURANTS_API_BUMP_TOKEN: ${{ secrets.RESTAURANTS_API_BUMP_TOKEN }}
+          SKALEET_BUMP_TOKEN: ${{ secrets.SKALEET_BUMP_TOKEN }}
+          SLACK_API_BUMP_TOKEN: ${{ secrets.SLACK_API_BUMP_TOKEN }}
+          TALEEZ_BUMP_TOKEN: ${{ secrets.TALEEZ_BUMP_TOKEN }}
```
Comment (paulRbr) [inline .github/workflows/deploy.yml]: Nice thanks a lot for this! Did you add the secret variables on the repo settings taken from  the token values of each APIs on Bump.sh?

**Review by paulRbr — APPROVED** [review]
I left a small questions about the Bump tokens, whether you have added some to the github settings or not? But LGTM, thanks alot!

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (vvenance) [reply .github/workflows/deploy.yml]: Yes they are all added using the right variable name formatting

**Pull request merged**